docs/serving/samples/knative-routing-go/index.md

```diff
@@ -260,6 +260,39 @@ The Gateway proxy checks the updated host, and forwards it to `Search` or
 
 ![Object model](./images/knative-routing-sample-flow.png)
 
+## Using internal services and `"httpProtocol": "Redirected"`
+
+Using the above approach, services will be available using two entrypoints into the cluster:
+The original ones provided by KNative Serving (`search-service.default.example.com` and `login-service.default.example.com`),
```

Comment:
```suggestion
The original ones provided by Knative Serving (`search-service.default.example.com` and `login-service.default.example.com`),
```

I think we use `Knative` instead of `KNative` so it should be consistent.